test/functional/mempool_accept.py

```diff
@@ -58,38 +56,32 @@ def check_mempool_result(self, result_expected, *args, **kwargs):
 
     def run_test(self):
         node = self.nodes[0]
+        miniwallet = MiniWallet(node)
 
         self.log.info('Start with empty mempool, and 200 blocks')
         self.mempool_size = 0
+        miniwallet.generate(10)
+        node.generate(190)
```

Comment:
Assuming this is done to mature the coinbases... is there any reason to do more than 100?

Reply:
Not really, just wanted to keep 200 blocks to mimic the current code